src/MSAL.PCL/Internal/Cache/TokenCacheKey.cs

```diff
@@ -77,8 +71,6 @@ public override string ToString()
             stringBuilder.Append(MsalHelpers.Base64Encode(ClientId) + "$");
             // scope is treeSet to guarantee the order of the scopes when converting to string.
             stringBuilder.Append(MsalHelpers.Base64Encode(Scope.AsSingleString()) + "$");
```

Comment:
the reason we base64 encoded the key was because of displayble id. We're removing the displayble id from the key, do we still need to do base64 encode?